### DISCUSSION TITLE
网页快捷键部分【非播放器的快捷键】

### DISCUSSION THREAD

**Discussion by nyano** [the1812/Bilibili-Evolved#2627] [功能建议 / Ideas]
是否可以在快捷键扩展中添加网页的快捷键删除或修改控制。即非视频播放器本身的快捷键
如：W：投币 ，D关闭弹幕等
我注意到某些国内浏览器具有全部关闭浏览器快捷键的功能，但是会把ctrl+w等快速关闭标签页等全部禁用掉。
这些网页中的快捷键仍然容易被误按。比如说关闭弹幕，总是误按后感觉为啥这视频没弹幕。。。半天后才意识到被自己关了。。。
谢谢

**Comment by Geeyun-JY3** [comment]
重复：#2474

是可以的。
V1 参考：https://github.com/the1812/Bilibili-Evolved/blob/master/src/video/keymap/key-bindings.md
V2 参考：https://github.com/the1812/Bilibili-Evolved/blob/preview/registry/lib/components/utils/keymap/help.md

**Reply by nyano** [reply]
收到。实测V2中【无动作】可以设置 多个按键。如设置为：q w e r t a s d f g 可将这些按键全部屏蔽掉
感谢

**Reply by the1812** [reply]
如果是脚本提供的快捷键, 直接把对应动作那里设个空也可以, 不用加到 `无动作` 里
